build.gradle

```diff
@@ -11,25 +11,25 @@ apply plugin: 'net.minecraftforge.gradle.forge'
 //Only edit below this line, the above code adds and enables the necessary things for Forge to be setup.
 
 
-version = "0.3.6c"
-group = "com.clussmanproductions.trafficcontrol" // http://maven.apache.org/guides/mini/guide-naming-conventions.html
-archivesBaseName = "trafficcontrol"
+version = "1.0"
+group = "com.yourname.modid" // http://maven.apache.org/guides/mini/guide-naming-conventions.html
+archivesBaseName = "modid"
```

Comment:
Looks like this is wiping out information.  Probably not intentional?

Reply:
Fixed. Not intentional